curriculum/challenges/english/25-front-end-development/quiz-recursion/66edd43cded6bff30944b676.md

```diff
@@ -17,439 +17,507 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What is recursion in programming?
 
 #### --distractors--
 
-Placeholder distractor 1
+A programming technique where a function calls itself.
```

Comment:
You can't have a distractor be the same as the answer, change this

Reply:
This comment should not have been resolved. The issue is still present.